tests/Unit/Libraries/Module/ModuleManagerTest.php

```diff
@@ -82,30 +102,17 @@ public function tearDown(): void
         }
 
         if($this->fs->isDirectory($apiModulePath)) {
-            $this->deleteDirectory($apiModulePath);
-        }
-    }
-
-    private function deleteDirectory(string $dir)
-    {
-        if (!is_dir($dir)) {
-            return;
-        }
-
-        $files = array_diff(scandir($dir), array('.', '..'));
-
-        foreach ($files as $file) {
-            $path = $dir . DS . $file;
-            is_dir($path) ? $this->deleteDirectory($path) : unlink($path);
-        }
-
-        if ($dir != uploads_dir()) {
-            rmdir($dir);
+            deleteDirectoryWithFiles($apiModulePath);
         }
     }
 
     private function getModuleConfigs(){
         return $this->fs->require($this->modulesConfigPath);
     }
 
+    private function getModuleManager($name = 'Api', $template = "DefaultApi"): ModuleManager
```

Comment:
why this method required, it's just contains one operation which can be called as it is